reactive/INTRODUCTION_EN.md

```diff
@@ -0,0 +1,153 @@
+# Introduction to Reactive Programming
+
+> Written by [A~Z](https://github.com/AZ-0)
+
+Do you know of reactive programming? No? You should!
+
+Let's begin with wikipedia's obscure definition:
+
+## Definition
+
+The original article has the [french](https://fr.wikipedia.org/wiki/Programmation_r%C3%A9active) definition as well, but seeing as it is quite useless I just removed it.
```

Comment:
```suggestion
The original article has the [french](https://fr.wikipedia.org/wiki/Programmation_r%C3%A9active) definition as well, but after judging that it was quite irrelevant, I just removed it.
```

Couldn't quite understand the `but seeing as it is quite useless`, I suggested an alternative